src/pages/messages/components/TicketList.tsx

```diff
@@ -53,13 +59,16 @@ export const TicketList = ({ ticketsQuery, setRowsPerPage, rowsPerPage, setPage,
         <title>{title}</title>
       </Helmet>
 
-      <Typography variant="h2" sx={{ mb: '1rem' }}>
-        {title}
-      </Typography>
-
-      <Box sx={{ mb: '1rem', display: 'flex', gap: '0.5rem' }}>
-        <SearchForm sx={{ flex: '1 0 15rem' }} placeholder={t('tasks.search_placeholder')} />
-      </Box>
+      <Grid container columns={16} gap={2} sx={{ px: { xs: '0.5rem', md: 0 } }}>
+        <Grid item xs={16}>
+          <SearchForm placeholder={t('tasks.search_placeholder')} />
+        </Grid>
+        <Grid item xs={16} md={5} lg={4}>
+          {isOnTasksPage && (
```

Comment:
Liten heads-up: frykter det vil bli en del logikk her i fremtiden, siden denne komponenten brukes for både Min Side og Oppgaver. Etter hvert som vi får inn flere filtre her som bare skal vises på en av sidene blir det fort en del ekstra logikk. Men syns likevel det gir mening å fortsette sånn som dette for nå, så får vi ha det i bakhodet å snu tidsnok om det blir et problem :)

Reply:
Ja, det gikk en liten varsellampe allerede med denne ene komponenten. Spent når det blir flere! Men som du sier, prøver for nå, og så endrer jeg når det blir for mye køl 👍